lib/active_fedora/relation/finder_methods.rb

```diff
@@ -198,7 +197,11 @@ def class_to_load(resource, cast)
         ActiveFedora::Base
       else
         # The true class may be a subclass of @klass, so always use from_class_uri
-        Model.from_class_uri(has_model_value(resource)) || ActiveFedora::Base
+        resource_class = Model.from_class_uri(has_model_value(resource)) || ActiveFedora::Base
```

Comment:
Do you think some folks might like to override the `ActiveFedora::Base` disjunct here?  If not, nothing to see here.  If so:

```ruby
# ...
resource_class = Model.from_class_uri(has_model_value(resource)) || default_class
# ...

def default_class
  ActiveFedora::Base
end
````